docs/guide/plotting.rst

```diff
@@ -200,41 +200,6 @@ then used to modify the plot:
     plt.colorbar()
     plt.show()
 
-It is possible to create the same plot, explicitly not using
-`~astropy.visualization.wcsaxes`, however, this will not have the features of
-`~astropy.visualization.wcsaxes` which include correct representation of
-rotation and plotting in different coordinate systems.
```

Comment:
If we plan to not support non-wcsaxes, probably don't need this example anymore.

Reply:
Well, people might want to do this still, perhaps we could point them at something in the example gallery here instead?

Reply:
(This example should have been purged before 3.0 tho)

Reply:
Do we have one?

Reply:
The example I updated in https://github.com/sunpy/sunpy/pull/5284 doesn't use `WCSAxes`.

Reply:
Added a ref to it.